Directory.Build.targets

```diff
@@ -23,4 +23,34 @@
       <Private>false</Private>
     </ProjectReference>
   </ItemDefinitionGroup>
+
+  <!-- WORKAROUND: Upgrading Microsoft.VisualStudio.Threading causes CreatePkgDef fail to run
+       because it is using runtime reflection to load a reference assembly.
+       This is due to a dependency on Bcl.Interfaces package references.  -->
+  <Target Name="GeneratePkgDef"
```

Comment:
You should rebase/merge in latest as this workaround is now in Workaround.targets and is no longer needed here.